### PR TITLE
DIvanyshyn - Task 1

### PR DESCRIPTION
**Pull request by DmytrIvd**
Я не сильно зрозумів 13 завдання

### CODE REVIEW

**Comment by m96roman** [comment]
> Я не сильно зрозумів 13 завдання. 
Впринципі ок. Тільки розкоментуй код. Мож закоментувати лише виклики функцій.

File: Bootcamp/HelperMethods/Extensions.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        /// <summary>
+        /// Swaps chars with specified indexes
+        /// </summary>
+        /// <param name="var">string char of which need to be swapped</param>
+        /// <param name="index1">index of first element</param>
+        /// <param name="index2">index of secound element</param>
+        /// <returns>New string with swapped chars</returns>
+        /// <exception cref="IndexOutOfRangeException">If indexes are out range</exception>
+        /// 
+        public static string swap(this string var, int index1, int index2)
```
Comment (olehkrut) [inline Bootcamp/HelperMethods/Extensions.cs:18]: all method names should begin with a capital letter, Swap not swap.

File: Bootcamp/HelperMethods/Reader.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace HelperMethods
+{
+    public static class Reader
```
Comment (olehkrut) [inline Bootcamp/HelperMethods/Reader.cs:5]: code style suggestion: use NewLine between methods and before return statement

File: Bootcamp/Homework Ivanyshyn/Program.cs [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+namespace Homework_Ivanyshyn
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //1. Create variable and initialize it with any value. Write into console variable name and it's value using nameof operator and interpolation.
+            //var str = new string('~', 100);
+
+            //Console.WriteLine($"Long string {str} something on end");
```
Comment (olehkrut) [inline Bootcamp/Homework Ivanyshyn/Program.cs:15]: nameof isn't used )

File: Bootcamp/Homework Ivanyshyn/Program.cs [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            //for (int i = i1; i <= i2; i++)
+            //{
+
+            //    if (i % 8 == 0)
+            //    {
+            //        Console.WriteLine(i);
+            //    }
+            //}
+            //13.Create enum type for TV channels(5 is enough). Write a switch statement that provides channel avarage rating.
+            //Dont clearly understood
```
Comment (olehkrut) [inline Bootcamp/Homework Ivanyshyn/Program.cs:117]: you did it write, so it's ok, well done!

File: Bootcamp/Homework Ivanyshyn/Program.cs [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+            //{
+            //    Console.WriteLine("Error");
+            //    return;
+            //}
+            //for (int i = i1; i <= i2; i++)
+            //{
+
+            //    if (i % 8 == 0)
+            //    {
+            //        Console.WriteLine(i);
```
Comment (olehkrut) [inline Bootcamp/Homework Ivanyshyn/Program.cs:113]: you also might optimize it a bit, just do `i += 8` after the `Console.WriteLine`

**Review by olehkrut — COMMENTED** [review]
Overall - everything is fine, well-done, I've added only minor suggestions

**Review by m96roman — COMMENTED** [review]
Good job!

File: Bootcamp/Homework Ivanyshyn/Program.cs [hunk 6]
```diff
@@ -10,117 +10,188 @@ class Program
...
+            while (i1 <= i2)
+            {
+                if (i1 % 8 == 0)
+                {
+                    Console.WriteLine(i1);
+                    i1 += 8;
+                    continue;
+                }
+                i1++;
+            }
```
Comment (DmytrIvd) [inline Bootcamp/Homework Ivanyshyn/Program.cs:98]: Changed algorithm a little

File: Bootcamp/Homework Ivanyshyn/Program.cs (hunk 5 above)
Comment (DmytrIvd) [reply Bootcamp/Homework Ivanyshyn/Program.cs]: All issues fixed in last commit

**Review by olehkrut — CHANGES_REQUESTED** [review]
pls move your homework task to the appropriate folder: `Bootcamp/Lecture1/Homework/YourName/` in doing so you'll have to update the solution file by removing old project and adding the existing one from new location, let me know in case of any problems

**Comment by DmytrIvd** [comment]
> pls move your homework task to the appropriate folder: `Bootcamp/Lecture1/Homework/YourName/` in doing so you'll have to update the solution file by removing old project and adding the existing one from new location, let me know in case of any problems

Fixed this issue

**Review by olehkrut — APPROVED** [review]

**Pull request merged**